src/ComputedAttribute/__init__.py

```diff
@@ -1,5 +1,4 @@
-from ExtensionClass import Base
-from ExtensionClass import C_EXTENSION
+from ExtensionClass import C_EXTENSION, Base
```

Comment:
The `isort` rules we have everywhere else specify one import per line.

Reply:
This is fixed in `meta/config`, I just have to re-apply it.